bundles/core/src/main/java/com/adobe/cq/wcm/core/components/services/link/PathProcessor.java

```diff
@@ -64,4 +67,13 @@
      * @return the external link of the given path
      */
     @NotNull String externalize(@NotNull String path, @NotNull SlingHttpServletRequest request);
+
+
+    /**
+     * Process the HTML attributes for the {@link com.adobe.cq.wcm.core.components.internal.link.LinkHandler}
```

Comment:
I would rephrase it as follows:
Processes HTML attributes and returns the processed attributes.
This is an extension point to modify HTML attributes, remove or add new ones.